### DISCUSSION TITLE
Sudo so-status keeps most services at WAIT_START

### DISCUSSION THREAD

**Discussion by iqworks** [Security-Onion-Solutions/securityonion#8880] [Unsupported Versions] [answered]
Hi everyone, 
    I am using :
    SO version 22.04.
    I am trying to use SO on my windows 10 laptop, I don’t use the cloud.
    I used securityonion-2.3.160-20220829.iso
    I used Centos 7
    For nodes, I just have laptops, TV’s and a printer.
    I am using a dell laptop and windows 10 version 10.0.19044 build 19044

  For two months my security onion VM in VMware has completely started (OK)  all services and I could get to the SOC. But, yesterday it stopped at the below image, no matter how many so-status commands I applied, only 4 services go to OK. 
![image](https://user-images.githubusercontent.com/5569263/194603481-53bb87c3-e237-404f-9426-2a962da9fdf1.png)
And when I let it just sit there, eventually I get the following error messages
![image](https://user-images.githubusercontent.com/5569263/194607832-605fe91e-7ad5-4f69-9365-a07a75fa10dc.png)
The only thing I can think of is that perhaps over the past two months, I could have had a file(s) that should be cleared out periodically? My .VMDK single file is 45.00 GB!!
   I also hear periodic beeps while the above print screen sits there. 
   I tried compacting the VMDK. I also tried defragging the VMDK. Now its 23.00 GB.
   I googled deleting the VMDK, it looks like under certain standards. But it looks unwise to delete the VMDK 
  Even after compacting and defragging, but the it still stops at the above print.
 
   When I do a  salt-call state..highstate,  I get this ..... 
![image](https://user-images.githubusercontent.com/5569263/194612387-8b211e71-5030-4930-8aa1-21a231abfa09.png)

 This makes me think it is running out of memory?
292.653434 nmi watchdog bug soft lockup  

  Thanks for any suggestions and advice

**Comment by reyesj2** [comment, marked as answer]
Is it possible you changed the hardware of the VM?

I would reinstall from ISO especially if you're losing more and more files. May be more time efficient to build a fresh version. For future builds you may try using the snapshot feature, so you can roll-back to a good running state.

**Reply by iqworks** [reply]
No, I didnt do anything. But its been working for 2 months. I will build a new one and then take a snap shot. 

i am wondering what this is and if the master.log is called only master/
**Local: 
  Data failed to compile:
Pillar failed to render with the following messages
Rendering SLS ‘logstash.nodes’ failed. Please see master log for details**

thank you, will keep you posted

**Reply by iqworks** [reply]
Thanks Reyesj2, 
you where correct, Some how the virtual network editor lost my settings. But I imported them from a saved copy and everything worked.
![image](https://user-images.githubusercontent.com/5569263/194651151-9abd23e7-8058-4f35-856b-f91c895b615c.png)
I just dont know how I lost those settings.

Thanks again
